hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -387,12 +388,24 @@ public List<ContainerInfo> listContainer(long startContainerID, int count)
     return listContainer(startContainerID, count, null, null, null);
   }
 
+  @Override
```

Comment:
IMO,
We should not provide new method, need update existing in such  a way works with both old and new for below scenario,

1) New Client - New Server
2) New Client - Old Server (since proto field is optional, so if old server, has***() check will confirm if field exit or not and can assign default as -1 as not exist)
3) Old Client - New Server (client will ignore new field)